packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m

```diff
@@ -39,6 +39,25 @@ - (dispatch_queue_t)methodQueue
     }];
 }
 
+RCT_EXPORT_METHOD(getIdentities: (nonnull NSDictionary*) AEPIdentityMap resolve:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```

Comment:
signature should be `getIdentities (RCTPromiseResolveBlock) resolve rejecter:(RCTPromiseRejectBlock)reject)`

Reply:
I will fix this in the iOS PR